HomeTask8/app/src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit,ViewChild,ElementRef, AfterViewInit } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
+})
+export class FooterComponent implements AfterViewInit {
+  @ViewChild('footertext', { static: false }) myfootertext: ElementRef;
+  @ViewChild('footer', { static: false }) myfooter: ElementRef;
+
+  constructor() { }
+
+  ngAfterViewInit() {
```

Comment:
is there any specific requirement for implementing the innerHTML here. If no then we can use the template as static and this component is not going to change in terms of functionality. If you are exploring then leave it as it is

Reply:
Added it while exploring.